wagtail/admin/panels/model_utils.py

```diff
@@ -62,6 +63,11 @@ def expand_panel_list(model, panels):
         if isinstance(panel, Panel):
             result.append(panel)
 
+        elif isinstance(panel, PanelPlaceholder):
+            real_panel = panel.construct()
+            if real_panel:
+                result.append(real_panel)
```

Comment:
Good opportunity to use a walrus operator :=)

```suggestion
            if real_panel := panel.construct()
                result.append(real_panel)
```

Reply:
Thanks! I very nearly did go for a walrus operator here, but it seemed slightly too unfamiliar - all the more reason to use it more and become more familiar with it though :-)